webpack/scenes/ContentViews/Details/Repositories/ContentViewRepositories.js

```diff
@@ -217,8 +231,37 @@ const ContentViewRepositories = ({ cvId, details }) => {
     return getContentViewRepositories(cvId, allParams, statusSelected);
   }, [cvId, repoTypes, statusSelected, typeSelected]);
 
-  const emptyContentTitle = __("You currently don't have any repositories to add to this content view.");
-  const emptyContentBody = __('Please add some repositories.'); // needs link
+  const noResults = (results && results.length === 0) && !searchQuery && status === STATUS.RESOLVED;
+  const emptyContentOverride =
+    noResults && statusSelected === ADDED;
+  const trulyEmpty =
+    noResults && statusSelected === ALL_STATUSES;
+  const emptyContentTitles = {
+    addRepos: __('No repositories have been added to this content view yet.'),
+    trulyEmpty: __('You currently don\'t have any repositories to add to this content view.'),
+  };
+  const emptyContentBodies = {
+    addRepos: __('Click to see repositories available to add.'),
+    trulyEmpty: __('Enable some Red Hat repositories or create a custom product.'),
+  };
+
+  const showPrimaryAction = emptyContentOverride || trulyEmpty;
+  const primaryActionButton = emptyContentOverride ? (
+    <Button ouiaId="empty-state-primary-action-button" onClick={handleShow}>
+      {__('Show repositories')}
+    </Button>
+  ) : (
+    <Button ouiaId="empty-state-primary-action-button">
+      <a href="/redhat_repositories" style={{ color: 'white', textDecoration: 'none' }}>{__('Enable Red Hat repositories')}</a>
+    </Button>
+  );
+
+  const secondaryActionLink = trulyEmpty ? ('/products/') : undefined;
```

Comment:
Should this go to `/products/` or `/products/new`? I changed it to products because what if you've created a product already but haven't added repos to it. But I'm on the fence.

Reply:
`/products/` is good. It has it's own empty state in bastion.. 😝